ppgen.py

```diff
@@ -4528,7 +4532,7 @@ def pm_guts(line, line_num):
           while i < len(self.wb) - 1 and self.wb[i].endswith("\\"):
             if self.wb[i+1].startswith(".pn") or self.wb[i+1].startswith(".bn"):
               self.crash_w_context(".pn or .bn not allowed within a continued dot directive", i)
-            elif self.wb[i+1].startswith(".") and re.match("\.[a-z]", self.wb[i+1]):
```

Comment:
I didn't exhaustively review this PR, but wouldn't this edit change the fundamental nature of the regex? There's a big difference between `\.` and `.`